src/bp/ui-studio/src/web/components/Layout/StatusBar.jsx

```diff
@@ -16,66 +20,81 @@ const ActionItem = props => (
         <div>
           <strong>{props.title}</strong>
         </div>
+        {props.shortcut && <div className={style.shortcut}>{props.shortcut}</div>}
         {props.description}
       </Tooltip>
     }
   >
-    <li className={style.statusBar__list__clickable}>{props.children}</li>
+    <div
+      className={classNames(style.clickable, style.item, props.className)}
+      {..._.omit(props, ['title', 'description', 'children', 'className'])}
```

Comment:
I won't complain about this line but I'm sure some would ...

Reply:
I would :)